### DISCUSSION TITLE
Methane GWP for the IPCC Method

### DISCUSSION THREAD

**Discussion by m-jamieson** [USEPA/LCIAformatter#118] [Q&A] [answered]
Assuming I did this correctly, I get an AR6, 100-yr characterization factor 27.9, which aligns with the value in the data file. The 27.9 doesn't match up with any of the values provided in [Table 7.15 of AR6](https://www.ipcc.ch/report/ar6/wg1/chapter/chapter-7/#7.6.1.1):

- CH4-fossil 29.8 +/- 11
- CH4-non fossil 27.0 +/- 11

Where did the 27.9 factor come from?

Secondly, I guess is the matter for which value gets mapped to FEDEFL emission/air/methane (for example). The AR5, 100-yr factor for methane in my mapping is 28, which suggests the "non-fossil" factor is being used there. So for AR6, would we expect the same mapping if there were separate fossil and non-fossil factors for methane?

**Comment by bl-young** [comment, marked as answer]
We are using data from Table 7.SM.7 (pdf [here](https://www.ipcc.ch/report/ar6/wg1/downloads/report/IPCC_AR6_WGI_Chapter07_SM.pdf)), which was pulled directly from [here](https://github.com/IPCC-WG1/Chapter-7/blob/main/data_output/7sm/metrics_supplement_cleaned.csv). The difference seems to mostly stem from:
> Chemical effects of CH4 and N2O are included ([Section 7.6.1.3](https://www.ipcc.ch/report/ar6/wg1/chapter/chapter-7/#7.6.1.3)) in Table 7.15

You are correct that we did defer to the non-fossil factor for methane. see https://github.com/USEPA/LCIAformatter/issues/97

I welcome @Andrew809 to chime in here on the choice of table for these data.

**Comment by bl-young** [comment]
if you are curious, the raw data was compiled into a single csv [here](https://github.com/USEPA/LCIAformatter/blob/master/lciafmt/data/IPCC_GWP_values.csv). We could probably do a better job linking those back to the specific tables used in each case.

**Comment by m-jamieson** [comment]
Thanks for pointing out the difference and the csv version! I'm sure it would've been helpful to us before now. I welcome whatever other information you can provide on the choice of table (factor) here.  The 7.SM.7 looks like it's the GWP from a pure radiative forcing perspective, and I think makes sense if your goal is to provide a single value for methane, regardless of source.

FWIW, for AR5, I believe we were one of the few organizations using a 100-yr GWP value of 36 for methane, which included the climate carbon feedbacks and the +2 adder for fossil methane oxidation given that the vast majority of methane emissions in our life cycle models are from fossil sources (coal mines and natural gas).
